openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -47,6 +47,13 @@ def execute(self):
 
         imageio_flame = project_settings["flame"]["imageio"]
 
+        # get host imageio settings enabled key if exists
```

Comment:
I suppose this is resolved?

Reply:
I don't believe so but if you all understand why if key isn't found is set to True to ensure backwards compatible it might be just me missing some context

Reply:
> I don't believe so but if you all understand why if key isn't found is set to True to ensure backwards compatible it might be just me missing some context

Ah I see, you're referring to this comment?

```python
        # if key was not found, set to True
        # ensuring backward compatibility
```

Also not sure why that's the case either.

Reply:
yeah exactly, we should expand the docstring to explain it then